### PR TITLE
Sprint 1/issue 16#redesign the events crud forms

### PR DESCRIPTION
**Pull request by DevLuis123**
Added styles for CRUD parts, buttons, and textboxes

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -9,5 +9,5 @@
     </ul>
 </nav>
 <h1>
-    Event managment system
+    Events management system
 </h1>
```
Comment (MauricioNC) [inline app/views/home/index.html.erb]: Add a blank line at the end of your code

File: app/views/events/show.html.erb [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
-<%= button_to 'Delete event', event_path(@event), style: "margin-bottom: 10px", 
-method: :delete, data: {turbo_confirm: "Are you sure?"} %> <%= link_to 'Back', events_path %>
+
+<%= button_to 'Edit event', edit_event_path(@event), method: :get, class: 'btn-1' %>
+<div class="">
+<%= button_to 'Delete event', event_path(@event), class: 'btn-2', 
+method: :delete, data: {turbo_confirm: "Are you sure?"} %> 
+</div>
+<%= button_to 'Back', events_path, method: :get, class: 'btn-3' %>
+</div>
```
Comment (MauricioNC) [inline app/views/events/show.html.erb]: Add a blank line at the end of your code

File: app/views/events/_event.html.erb [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-  <div style="margin-bottom: 5px"><b>Event description:</b> <%= event.description %></div>
-  <div style="margin-bottom: 5px"><b>Event date:</b> <%= event.date %></div>
-  <div style="margin-bottom: 5px"><b>Event location:</b> <%= event.location %></div>
-  <div style="margin-bottom: 15px"><b>Event price:</b> <%= event.price %></div>
+<div class= "grid grid-flow-row gap-3">
+  <div><b class="b">Event title:</b> <%= event.title%></div>
+  <div><b class="b">Event description:</b> <%= event.description %></div>
+  <div><b class="b">Event date:</b> <%= event.date %></div>
+  <div><b class="b">Event location:</b> <%= event.location %></div>
+  <div><b class="b">Event price:</b> <%= event.price %></div>
```
Comment (MauricioNC) [inline app/views/events/_event.html.erb]: If you are using tailwind, why don't you apply the font styles directly in the <b></b> tags?

File: app/views/events/new.html.erb [hunk 4]
```diff
@@ -2,4 +2,4 @@
 <div>
   <%= render 'form', event: @event %>
 </div>
-<%= link_to 'Back', events_path %>
+<%= button_to 'Back', events_path, method: :get, class: 'btn-3 ml-5' %>
```
Comment (MauricioNC) [inline app/views/events/new.html.erb]: Use link_to for get requests, and button_to for delete and post requests

File: app/views/events/edit.html.erb [hunk 5]
```diff
@@ -1,3 +1,3 @@
 <h1>Edit event</h1>
 <%= render 'form', event: @event %>
-<%= link_to 'Back', event_path(@event) %>
+<%= button_to 'Back', events_path, method: :get, class: 'btn-3 ml-5' %>
```
Comment (MauricioNC) [inline app/views/events/edit.html.erb]: Use link_to for get requests, and button_to for delete and post requests

File: app/views/events/_form.html.erb [hunk 6]
```diff
@@ -6,31 +6,36 @@
...
+  </div> 
+
+  <%= button_tag(type: 'submit', class: 'btn-2') do %>
+  Send event
+  <% end %>
   </div> 
-    
-  <%= form.submit style: "margin-bottom: 15px" %> 
+  
+
```
Comment (MauricioNC) [inline app/views/events/_form.html.erb]: Remove extra empty lines

File: test/events_controller_test.rb [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    assert_difference('Event.count') do
+      post :create, params: { event: {
+                                        title: 'New Event',
+                                        description: 'Description',
+                                        date: '2024-01-25',
+                                        location: 'Location',
+                                        price: 10
+                                        }
+                                    }
+end
```
Comment (MauricioNC) [inline test/events_controller_test.rb]: Indent correctly this code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by juperameza-lizos — CHANGES_REQUESTED** [review]
Take care of mauricio comments

File: app/views/home/index.html.erb (hunk 1 above)
Comment (ErikGmz) [reply app/views/home/index.html.erb]: Done!

File: app/views/events/_event.html.erb (hunk 3 above)
Comment (ErikGmz) [reply app/views/events/_event.html.erb]: Fixed!

File: app/views/events/edit.html.erb (hunk 5 above)
Comment (ErikGmz) [reply app/views/events/edit.html.erb]: Done!

File: app/views/events/show.html.erb (hunk 2 above)
Comment (ErikGmz) [reply app/views/events/show.html.erb]: Done!

File: app/views/events/_form.html.erb (hunk 6 above)
Comment (ErikGmz) [reply app/views/events/_form.html.erb]: Fixed!

**Review by juperameza — APPROVED** [review]
LGTM
Used my work account for the other comment lol
IF you could please delete that comment

File: test/events_controller_test.rb (hunk 7 above)
Comment (ErikGmz) [reply test/events_controller_test.rb]: This file was supposed to be in another branch, so it was deleted in this one and fixed in the other one.

**Review by MauricioNC — APPROVED** [review]
LGTM!

**Pull request merged**